src/webapi/document.rs

```diff
@@ -58,6 +58,19 @@ impl Document {
         }
     }
 
+    /// Creates an element with the specified namespace URI and qualified name.
+    /// To create an element without specifying a namespace URI, use the `createElement` method.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElementNS)
+    // https://dom.spec.whatwg.org/#ref-for-dom-document-createelementns
+    pub fn create_element_ns( &self, namespace_uri: &str, tag: &str ) -> Result< Element, TODO > {
```

Comment:
Thanks for the PR!

Since you're adding this - could you replace the `TODO` with an actual error type? (The `TODO`s were added in batch to mark a bunch of already existing methods as throwing, but we want to get rid of them eventually.)

I can see from the specs that this can throw at least `NotSupportedError`, `NamespaceError` and `InvalidCharacterError`. So:
1) please create an aggregate error with the `error_enum_boilerplate!`,
2) if possible (and it should be possible from what I can see) please add tests for these errors (the specs say in what situations they get thrown; just pick one per error to cover every possible variant),

Reply:
I didn't see a NotSupportedError for this file. Also added tests for createElement.